website/docs/Basic_operations/update.md

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_position: 4
+---
+
+
+# Update operation
+
+The update operation modifies a document record in a collection, based on a given query parameter and update.
+The `$set` operator assigns the updated record to the document.
+
+## Modify a single document
+
+Use the `updateOne()` method to update a single document in a collection.
+This operation filters a collection using a query parameter, and updates given fields within that document.
+
+```sh
+db.collection.updateOne({<query-params>}, {$set: {<update fields>}})
```

Comment:
Should we mention update operators here? We use them in the example but don't describe what operators are supported or how to use them. As that's basics we can do it briefly somehow.

Reply:
I've highlighted a few operators above  based on info [here](https://github.com/FerretDB/FerretDB/issues/229). Please let me know if there's any other I must have missed.